Formula/scrcpy.rb

```diff
@@ -1,8 +1,8 @@
 class Scrcpy < Formula
   desc "Display and control your Android device"
   homepage "https://github.com/Genymobile/scrcpy"
-  url "https://github.com/Genymobile/scrcpy/archive/v1.17.tar.gz"
-  sha256 "c16f1fe1789290d0c7dd3a2778f33d6dc6347ffe3e78e64127b85eda52420d7f"
+  url "https://github.com/Genymobile/scrcpy/archive/v1.18.tar.gz"
+  sha256 "6aef93291d24bb7977fb0991c4bd093f0c660e5d0aa881f7ac598b1b1ce9be3a"
   license "Apache-2.0"
   revision 1
```

Comment:
The `revision 1` should be removed here (`bump-formula-pr` does that automatically and intentionally).

Reply:
I wasn't sure if its intended 😄